SPECS/xcp-clipboardd.spec

```diff
@@ -0,0 +1,36 @@
+Name:           xcp-clipboardd
+Version:        1.0.2
+Release:        1
+Summary:        Daemon to share a virtualized Windows clipboard
+License:        GPLv3
+URL:            https://github.com/xcp-ng/xcp-clipboardd
+Source0:        https://github.com/xcp-ng/xcp-clipboardd/archive/v%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires:  cmake3
+BuildRequires:  make
+BuildRequires:  gcc
+BuildRequires:  xen-libs-devel
+
+Obsoletes:      xs-clipboardd
+
+%description
+Share clipboard between guest Windows and host with VNC.
+
+%prep
+%autosetup -p1
+
+%build
+mkdir build && cd build
+cmake3 .. -DCMAKE_INSTALL_PREFIX=%{buildroot}/opt/xensource
```

Comment:
Does cmake3 provide RPM macros that define common CFLAGS?

Reply:
Yes in `/usr/lib/rpm/macros.d/macros.cmake3`

Reply:
Packaging guidelines from linux distros encourage the use of such macros so that compilation flags defined at the distro level are propagated to all (or most) RPMs. Can you review the flags that are defined in our build environment and see what seems the best option here?

Reply:
```
# rpm --eval "%cmake3"

  CFLAGS="${CFLAGS:--O2 -g}" ; export CFLAGS ; 
  CXXFLAGS="${CXXFLAGS:--O2 -g}" ; export CXXFLAGS ; 
  FFLAGS="${FFLAGS:--O2 -g}" ; export FFLAGS ; 
  FCFLAGS="${FCFLAGS:--O2 -g}" ; export FCFLAGS ; 
   
  /usr/bin/cmake3 \
        -DCMAKE_C_FLAGS_RELEASE:STRING="-DNDEBUG" \
        -DCMAKE_CXX_FLAGS_RELEASE:STRING="-DNDEBUG" \
        -DCMAKE_Fortran_FLAGS_RELEASE:STRING="-DNDEBUG" \
        -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
        -DCMAKE_INSTALL_PREFIX:PATH=/usr \
        -DINCLUDE_INSTALL_DIR:PATH=/usr/include \
        -DLIB_INSTALL_DIR:PATH=/usr/lib64 \
        -DSYSCONF_INSTALL_DIR:PATH=/etc \
        -DSHARE_INSTALL_PREFIX:PATH=/usr/share \
%if "lib64" == "lib64" 
        -DLIB_SUFFIX=64 \
%endif 
        -DBUILD_SHARED_LIBS:BOOL=ON
```
It seems good.